### DISCUSSION TITLE
华南师范大学

### DISCUSSION THREAD

**Discussion by bobby285271** [CollegesChat/university-information#17] [University]
## 南海校区

> 报读软件工程或人工智能的话就是来男海（划掉）南海校区的啦。

**宿舍是上床下桌吗**
都是（4-5 人一室）。

**教室和宿舍有没有空调**
都有。

**有独立卫浴吗**
都有（特别地，宿舍 E 座为一个宿舍分 AB 两室各住四人，两室共用客厅和两个厕所隔间）。

**有早自习、晚自习吗**
软院大一有晚自习，其他年级无；其它专业不清楚。

**有晨跑吗**
没有。

**每学期跑步打卡的要求是多少公里，可以骑车吗**
南海校区特色，大一大二有要求，占体育成绩 10%，一学期约 18 次及格，约 30 次满分。
男生每次 2.5km，女生每次 1.5km，要求经过两个随机打卡点。
南海校区太小没人骑车，但是刷乐跑方式多样（咳咳）。

**寒暑假放多久，每年小学期有多长**
寒假约 4 周，暑假约 7 周，在广东高校放假时间算少的。
没有小学期。

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
允许，在宿舍楼下拿或五分钟路程去学校后门拿。

**学校附近有地铁站吗**
需要等佛山地铁 3 号线开通（至少 2023 年），校门有公交站可坐公交到附近商圈 / 高铁站 / 广州。
意味着周末想回广州 / 回南海校区的话要准备好跟整个校区乃至整个狮山大学城想回广州 / 回南海校区的人一起挤公交。

**宿舍楼有洗衣机吗**
没有公用洗衣机，自己宿舍买自己的。

**校园网怎么样**
勉强能用，经常抽风（饭堂无 wifi，部分教室 wifi 信号非常弱）。

**每天断电断网吗**
都不断。

**食堂价格贵吗，会吃出异物吗**
还行（南海校区只有一个食堂，菜式不是很多，不要试图跟其他校区的食堂对比）；不会。
另外隔壁东软学院的饭堂非常不错。

**洗澡热水供应时间**
只有晚上有。

**校园内可以骑电瓶车吗，电池在哪能充电**
校园很小（宿舍到教学楼仅 5 分钟步行），无需电瓶车。

**宿舍限电情况**
基本不限电。
软院会有每月一次的宿舍检查（会提前通知何时检查，要求检查时留人），检查内容包括内务和违禁电器（1000W+）。
查的严不严基本看检查的是谁（由学生检查），个人储物柜是不会开的，每个宿舍负责检查的同学一学年下来基本是固定的。
其它专业不清楚。

**通宵自习有去处吗**
宿舍每层有自习室（我没去过，不了解情况如何），E 座的同学可在客厅（雾）。

**大一能带电脑吗**
可以。

**学校里面用什么卡，饭堂怎样消费**
一卡通，用于饭堂（可实体卡也可直接手机扫码）、宿舍热水、体测。

**学校会给学生发银行卡吗**
要求有一张建行卡，没建行卡的入学后会组织统一办卡（用于发补助、奖学金和报销）。

**学校的超市怎么样**
只有一间超市，日常用品基本都有，价格还算合理。

**学校的收发快递政策怎么样**
校内有菜鸟驿站，但是有些快递不一定能进驿站（好像是百世和申通），后门外也有流动收取点。

**学校里面的共享单车数目与种类如何**
校园很小，校内用不着也没人用共享单车。
出校最近的商业广场 / 隔壁东软学院食堂从校门出发步行不超过 10 分钟。
除此之外感觉狮山大学城没别的地方好去了，再远的地方还是坐公交去吧。

**现阶段学校的门禁情况如何**
疫情期间出校请假要上系统请假辅导员批假（不是太离谱的理由基本都批）。
进出校给保安看请假审批记录（曾有自己写网页仿造的好像被发现了?）。

**宿舍晚上查寝吗，封寝吗，晚归能回去吗**
不查寝，理论上有封寝，过 23:30 还不回宿舍的最好跟辅导员打声招呼。
曾试过两点多回宿舍（有提前跟辅导员说），宿舍大门没锁，无宿管看门。

**Comment by YanWQ-monad** [comment]
> 补充一条群友的回答


**宿舍是上床下桌吗**
看校区和宿舍，石牌校区历史久远，差的宿舍真的很差。没有独立卫浴没有阳台。其他校区会好些。但是石牌也有四人间上床下桌的，一般女生宿舍学院抽签决定咯，大二的时候可能会重新抽一波签到好的宿舍。男生就别想了

**教室和宿舍有没有空调**
都有

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
看宿舍，没有的也是有隔间的，就在宿舍里面

**有早自习、晚自习吗**
看学院，计院，数科院大一有

**有晨跑吗**
无

**每学期跑步打卡的要求是多少公里，可以骑车吗**
不强制要求，但可以选拿[非正式课程学时](https://www.zhihu.com/question/315529252/answer/624608460) 。可以骑车

**寒暑假放多久，每年小学期有多长**
一般每学期19周左右，没小学期

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
疫情前楼下，疫情后校门

**学校附近有地铁站吗**
石牌西门100米有，石牌地理位置还是非常爽的

**宿舍楼有洗衣机吗**
看宿舍，独立卫浴的估计都有。没有的就只有公共几台

**校园网怎么样**
30/月，大部分时间还行，打游戏的同学说不行。接有线网会好很多

**每天断电断网吗，几点开始断**
不断

**食堂价格贵吗，会吃出异物吗**
丰俭由人，没听说有过

**洗澡热水供应时间**
看宿舍，主要看宿管。大部分6点到凌晨3，4点都有

**校园内可以骑电瓶车吗，电池在哪能充电**
石牌校区没有，主要是因为校区小&宿舍条件差吧

若有报考计算机学院的快逃。学院那点满滤镜和疯狂借位避重就轻的招生宣传看完就忘了吧。就是因为专业水平差才讲“人文计机”啦。客观的说211的头衔在网申阶段还是有点用的，但是教学和行政水平实在不敢恭维。

**Comment by Yibo2805** [comment]
有大学城校区和石牌的具体情况吗
